planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/ConfiguredFeature.java

```diff
@@ -95,24 +91,55 @@ public ConfiguredFeature(FeatureLayer layer, TagValueProducer tagValueProducer,
     }
     processors.add(makeFeatureProcessor(feature.minZoom(), Integer.class, Feature::setMinZoom));
     processors.add(makeFeatureProcessor(feature.maxZoom(), Integer.class, Feature::setMaxZoom));
-    if (layer.postProcess() == null) {
+    processors.add(makeFeatureProcessor(feature.tolerance(), Double.class, Feature::setPixelTolerance));
+    processors.add(makeFeatureProcessor(feature.toleranceAtMaxZoom(), Double.class, Feature::setPixelToleranceAtMaxZoom));
+
+    addPostProcessingImplications(layer, feature, processors, rootContext);
```

Comment:
I think we want to set feature tolerance after adding post processing implications so that if the value is set explicitly on a feature that takes precedence over any defaults set from post-processing config   https://github.com/onthegomap/planetiler/pull/1307#discussion_r2259921155

```suggestion

    addPostProcessingImplications(layer, feature, processors, rootContext);
    
    // per-feature tolerance settings should take precedence over defaults from post-processing config
    processors.add(makeFeatureProcessor(feature.tolerance(), Double.class, Feature::setPixelTolerance));
    processors.add(makeFeatureProcessor(feature.toleranceAtMaxZoom(), Double.class, Feature::setPixelToleranceAtMaxZoom));
```